bin/configs/spring-cloud-deprecated-oas3.yaml

```diff
@@ -0,0 +1,13 @@
+generatorName: spring
+library: spring-cloud
+outputDir: samples/client/petstore/spring-cloud-deprecated
```

Comment:
please add the new sample folder path to .github/workflows/samples-spring.yaml so that the CI can test the change moving forward.